pylint/checkers/strings.py

```diff
@@ -892,6 +899,55 @@ def process_non_raw_string_token(
             # character can never be the start of a new backslash escape.
             index += 2
 
+    @check_messages("possible-f-string-as-string")
+    def visit_const(self, node: astroid.Const):
+        self._detect_possible_f_string(node)
```

Comment:
```suggestion
        if node.pytype() == "builtins.str" and not isinstance(
            node.parent, astroid.JoinedStr
        ):
            self._detect_possible_f_string(node)
```
I would put the conditional here to diminish the indentation in the other function.